app/views/candidate_mailer/changed_offer.text.erb

```diff
@@ -1,13 +1,50 @@
 Dear <%= @application_form.first_name %>,
 
-<%= @course_option.course.provider.name %> has changed the details of your offer.
+# Offer changed by <%= @course_option.course.provider.name %>
 
-Contact <%= @course_option.course.provider.name %> directly if you have any questions about this.
+Your offer from <%= @course_option.course.provider.name %> to study <%= @course_option.course.name_and_code %> at <%= @course_option.site.name %> has been changed.
+
+<%= @offered_course_option.course.provider.name %> has offered you the following course:
+
+Provider: <%= @offered_course_option.course.provider.name %>
+Course: <%= @offered_course_option.course.name_and_code %>
+Location: <%= @offered_course_option.site.name %>
+Full time or part time: <%= @offered_course_option.course.study_mode.humanize %>
+
+The conditions of your offer have not been changed.
+
+If you were not expecting this change, contact <%= @course_option.course.provider.name %> to find out more.
+
+<% if @is_awaiting_decision %>
+
+# You can accept the offer or wait to hear back about other applications
+
+<%= render "wait_or_respond" %>
+
+<% else %>
 
-You can sign in to Apply for teacher training to check the status of your application(s):
+# Make a decision by <%= @application_choice.decline_by_default_at&.to_s(:govuk_date) %>
```

Comment:
I think there should be no need for an `&` here as all offered choices should have a `decline_by_default_at` and we'd want to fail loudly if one didn't — not least because this sentence wouldn't be complete!